test/units/modules/test_iptables.py

```diff
@@ -876,3 +876,39 @@ def test_insert_rule_with_wait(self):
             '-j',
             'ACCEPT'
         ])
+
+    def test_comment_position_at_end(self):
+        """Test flush without parameters"""
+        set_module_args({
+            'chain': 'INPUT',
+            'jump': 'ACCEPT',
+            'action': 'insert',
+            'comment': 'this is a comment',
```

Comment:
I think this should include some parameters that are problematic when the appear before `comment`.
```suggestion
            'cstate': ['NEW'],
            'comment': 'this is a comment',
```

Reply:
Fixed in https://github.com/ansible/ansible/pull/71496/files#diff-a184f210bf607737e23e4930d925ba5fR886.